.github/workflows/aws_copilot_deployment.yml

```diff
@@ -82,7 +82,7 @@ jobs:
 
   e2e_test:
     # Do not run these against the prod environment without addressing the auth/JWT self-signing done by e2e tests.
-    if: ${{ inputs.copilot_environment == 'test' && inputs.run_e2e_tests == true }}
+    if: ${{ inputs.run_e2e_tests == true && (inputs.copilot_environment == 'dev' || inputs.copilot_environment == 'test') }}
```

Comment:
Not sure if the syntax is correct. But happy to try it on prod. Looks plausible.

Reply:
You could always read the docs and check ;)

Reply:
Good one.. :rofl: